lib/utils.js

```diff
@@ -23,3 +25,10 @@ exports.getImageMimeType = function getImageMimeType (imagePath) {
       return undefined
   }
 }
+
+exports.isRevealTheme = function isRevealTheme (theme) {
+  if (fs.existsSync(path.join(process.cwd(), 'public', 'build', 'reveal.js', 'css', 'theme', theme + '.css'))) {
```

Comment:
We should use `__dirname` instead of process.cwd() to prevent getting wrong path.